lib/dateTime.js

```diff
@@ -3,73 +3,96 @@ import { getDocumentLocaleSettings, getLanguage, merge } from './common.js';
 const hour24locales = ['da', 'de', 'es', 'fr', 'nl', 'pt', 'sv', 'tr', 'zh'];
 const mondayFirstDayLocales = ['da', 'de', 'fr', 'nl', 'sv', 'tr'];
 
-// leaving out the following since do not throw and have multiple possible offsets: AMT, AST, BST, CDT, CST, ECT, GST, IST, LHST, MST, PST, SST
```

Comment:
Could you include that URL where you got this data in a comment? Just in case we need to go back in here later.